examples/dataset/speaker_diarization/README.md

```diff
@@ -65,3 +65,29 @@ optional arguments:
   --sample-count SAMPLE_COUNT
                         Number of samples to use, all samples are used if 0.
 ```
+
+## Quality Standards Guide
+
+Once the dataset and results have been uploaded to Kolena, visit [Kolena](https://app.kolena.io/redirect/) to
+[explore the data and results](https://docs.kolena.io/dataset/quickstart/#step-3-explore-data-and-results).
+
+Here are our [Quality Standards](https://docs.kolena.io/dataset/core-concepts/quality-standard/) recommendations for
```

Comment:
nit: I think we want `[Quality Standard]` instead of `[Quality Standards]`. Reads slightly better imo

Reply:
Nevermind, keep to be consistent with other examples

Reply:
Given that we are using `Quality Standards` in the UI, would singular still the preferred form here?

Reply:
We've landed "Quality Standards" (plural) 👍